source/Components/ListMenu.cpp

```diff
@@ -211,7 +211,7 @@ namespace FileShare::GUI::Components {
         size_t activeItemsSize = this->activeItems.size();
         size_t activeSubItemsSize = this->activeSubItems.size();
 
-        std::vector<const tgui::String> selection;
+        std::vector<tgui::String> selection;
```

Comment:
Curious - what was the reason for removing the const ? Was it causing issues ?
I dont think we would need to modify the strings of the selection, so would makes sense that they are const imo ?

I would even wonder if we could have `const &` cause then it would avoid a copy-construction